wagtail/admin/templates/wagtailadmin/shared/side_panels/preview.html

```diff
@@ -1,64 +1,100 @@
-                    </p>
+    <div class="w-preview__area">
+        <div class="w-preview__controls{% if has_multiple_modes %} w-preview__controls--multiple{% endif %}">
+            {% block error_banner %}
+                <div id="w-preview-error-banner" class="w-preview__error-banner">
+                    {% icon name='warning' classname='default' %}
+                    <div>
+                        <h3 class="w-preview__error-title">
+                            {% trans 'Preview is out of date' %}
+                        </h3>
+                        <p class="w-preview__error-details">
+                            {% trans 'Correct the validation errors to resume preview (saving will highlight errors)' %}
+                        </p>
+                    </div>
                 </div>
-            </div>
-            {% if has_multiple_modes %}
-                {% rawformattedfield label_text=_("Preview mode") id_for_label="id_preview_mode" classname="preview-panel__modes" %}
-                    <select id="id_preview_mode" name="preview_mode" class="preview-panel__mode-select" data-preview-mode-select>
-                        {% for internal_name, display_name in object.preview_modes %}
-                            <option value="{{ internal_name }}"{% if internal_name == object.default_preview_mode %} selected{% endif %}>{{ display_name }}</option>
-                        {% endfor %}
-                    </select>
-                {% endrawformattedfield %}
-            {% endif %}
+            {% endblock %}
+            {% block mode_select %}
+                {% if has_multiple_modes %}
+                    {% rawformattedfield label_text=_("Preview mode") id_for_label="id_preview_mode" classname="w-preview__modes" %}
+                        <select id="id_preview_mode" name="preview_mode" class="w-preview__mode-select" data-w-preview-target="mode" data-action="w-preview#setPreviewMode">
+                            {% for internal_name, display_name in object.preview_modes %}
+                                <option value="{{ internal_name }}"{% if internal_name == object.default_preview_mode %} selected{% endif %}>{{ display_name }}</option>
+                            {% endfor %}
+                        </select>
+                    {% endrawformattedfield %}
+                {% endif %}
+            {% endblock %}
         </div>
 
-        <div class="preview-panel__wrapper">
-            <iframe id="preview-iframe" loading="lazy" title="{% trans 'Preview' %}" class="preview-panel__iframe" data-preview-iframe aria-describedby="preview-panel-error-banner">
-                <div>
-                     {# Add placeholder element to support styling content when iframe has loaded #}
-                </div>
-            </iframe>
+        <div class="w-preview__wrapper">
+            {% block iframe %}
+                <iframe
+                    id="w-preview-iframe"
```

Comment:
Adding a `w-` prefix to the ID to follow our recent practices for singleton elements where the ID is significant to Wagtail.